scripts/breadcrumbs.js

```diff
@@ -86,7 +86,7 @@ export async function loadBreadcrumbs() {
   decorateBlockWithRegionId(breadcrumb, 'Hero|Breadcrumb');
 
   // check if breadcrumb div exists
-  if (breadcrumb !== undefined) {
+  if (breadcrumb !== undefined && breadcrumb !== null) {
```

Comment:
Can we simplify and catch all "nullish" values? -> `if (breadcrumb)`

Reply:
I reverted this, I saw that there is another PR that has this change